ru/part2.md

```diff
@@ -0,0 +1,109 @@
+просматривается явная закономерность. В последние годы количество данных,
+которые требуется передать, постепенно выросло до отметки 1,9Мб и выше, но, что
+наиболее важно в этом контексте, в среднем требуется загрузить сотню отдельных
+ресурсов, чтобы  отобразить каждую страницу.
+
+Как показывает график, тренд был растущим, но позднее нет никаких признаков,
+что он будет дальше меняться. Он показывает рост суммарного размера передачи
+(зелёным) и суммарного числа запросов (красным), используемых в среднем для
+обслуживания наиболее популярных сайтов в мире и как оно изменилось за
+последние четыре года.
+
+![transfer size growth](https://raw.githubusercontent.com/bagder/http2-explained/master/images/transfer-size-growth.png)
+
+## 2.5. Задержка убивает
+
+<img style="float: right;" src="https://raw.githubusercontent.com/bagder/http2-explained/master/images/page-load-time-rtt-decreases.png" />
+
+HTTP 1.1 очень чувствителен к задержкам, частично из-за того, что в конвейерной
+передаче HTTP по-прежнему хватает проблем и она отключена у подавляющего числа
+пользователей.
+
+В то время, как все мы наблюдали значительное увеличение доступной пропускной
+полосы у пользователей за последние несколько лет, мы не видели подобного
+уровня снижения задержки. Каналы с высокой задержкой, как у многих современных
+мобильных технологий, значительно снижают ощущение хорошей и быстрой
+веб-навигации, даже если у вас имеется действительно широкополсное подключение.
+
+Другой пример, когда действительно требуется низкая задержка, это некоторые
+виды видео, такие как видео-конференция, игры и подобное, где требуется
+передать не только заранее созданный поток.
+
+## 2.6. Блокировка начала очереди
+
+Конвейерная передача HTTP – это способ отправки очередного запроса,  при
+ожидании ответа на предыдущий запрос. Это похоже на очередь к кассиру в банке
+или супермаркете. Вы не знаете, что за люди перед вами: быстрые клиенты или
+надоедливые персоны, которым потребуется бесконечное время, чтобы завершить
+обслуживание – блокировка начала очереди.
+
+<img style="float: right;" src="https://raw.githubusercontent.com/bagder/http2-explained/master/images/head-of-line-blocking.jpg" />
+
+Безусловно вы можете тщательно выбирать очередь и в итоге выбрать ту, которую
+посчитаете правильной, а иногда вы можете создать свою собственную очередь, но,
+в  конце концов, вы не сможете избежать принятия решения и однажды выбрав
+очередь, вы не можете её сменить.
+
+Создание новой очереди связано с производительностью и расплатой ресурсами, и
+не может масштабироваться за пределы небольшого числа очередей. Для этой задачи
+нет идеального решения.
+
+Даже сегодня, в 2015 году, большинство веб-браузеров на десктопах поставляются
+с отключенным конвейером HTTP по умолчанию.
+
+Дополнительные сведения по этой проблеме могут быть найдены в [баг-трекере
+Firefox под номером
+2643541](https://bugzilla.mozilla.org/show_bug.cgi?id=264354).
```

Comment:
The bug entry number is 264354. You have an extra 1 at the end of the link title.